Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/States/InteractableStates.cs

```diff
@@ -95,7 +99,10 @@ public enum InteractableStateEnum
             new State(){ Index = 10, Name = "Gesture", ActiveIndex = -1, Bit = 0, Value = 0},
             new State(){ Index = 11, Name = "GestureMax", ActiveIndex = -1, Bit = 0, Value = 0},
             new State(){ Index = 12, Name = "Collision", ActiveIndex = -1, Bit = 0, Value = 0},
-            new State(){ Index = 13, Name = "Custom", ActiveIndex = -1, Bit = 0, Value = 0}
+            new State(){ Index = 13, Name = "VoiceCommand", ActiveIndex = -1, Bit = 0, Value = 0},
```

Comment:
Did you intend to also add the vice command here? It wasn't in the PR description, was this extra code that snuck in?

Reply:
This was a bug I noticed while adding the grip state, VoiceCommand was added previously to the enum above, but was never added to the states list.